Junk4Dollars-appUITests/Junk4Dollars_appUITests.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Junk4Dollars_appUITests.swift
+//  Junk4Dollars-appUITests
+//
+//  Created by John Riccardi on 10/28/19.
+//  Copyright © 2019 John Riccardi. All rights reserved.
+//
+
+import XCTest
+
+class Junk4Dollars_appUITests: XCTestCase {
+    func testHelloDefault() {
+        let app = XCUIApplication()
+        app.launch()
+        app.buttons["hello_btn"].tap()
+        let helloLabel = app.staticTexts["output_label"]
+        XCTAssert(helloLabel.exists)
+    }
+    
+    func testHelloName() {
+        let app = XCUIApplication()
+        app.launch()
+        let textInput = app.textFields["text_input"]
+        textInput.tap()
+        textInput.typeText("John")
+        app.buttons["hello_btn"].tap()
+        let helloJohnLabel = app.staticTexts["Hello, John!"]
+        XCTAssert(helloJohnLabel.exists)
+    }
+    
+    func testResetButton() {
```

Comment:
Test names should explain what the test is doing and looking for.
In this case, something like `testResetButtonClearsOutput` would be better.

Also, the reset button is doing something else that it's not being tested. What is it?

Reply:
Changed the name and added a test for `testResetButtonClearsInput`